Core/include/Acts/Geometry/CylinderPortalShell.hpp

```diff
@@ -102,27 +102,29 @@ class SingleCylinderPortalShell : public CylinderPortalShell {
 /// the stacking. Depending on the direction, portals on the shells of children
 /// are merged or fused. Subsequently, portal access respects shared portals
 /// between shells. Below is an illustration of a stack in the r direction:
-///
-///  Fused         +-----------------+
-/// portals ----+  |                 |
-///   |         |  v           OuterCylinder
-///   |  +------+------+
-///   |  |      |      |
-///   |  |      |      |<--+
-///   +--+---+  v      |   |
-///      +---+---------+   |
-///      |   |         |   |      Shared portal
-///      |   |         |<--+---      (grid)
-///      |   v         |   |      PositiveDisc
-///      +-------------+   |
-/// r ^  |             |   |
-///   |  |             |<--+
-///   |  |             |
-///   |  +-------------+       InnerCylinder
-///   +----->      ^            (if rMin>0)
-///          z     |                 |
-///                +-----------------+
-///
+/**
```

Comment:
I'm not sure I understand this, does this make a difference anywhere?

Reply:
It fixes the rendered ASCII [here](https://acts.readthedocs.io/en/latest/core/geometry/concepts.html#_CPPv4N4Acts24CylinderStackPortalShellE).

This is the before:
![image](https://github.com/user-attachments/assets/d17680b5-68f5-49dc-b81a-abec3b1f0982)

This is the after:
![image](https://github.com/user-attachments/assets/6e46e71e-240e-444d-817a-1f2b5b070c3b)


If you are referring to the `/**...*/` style instead of the `///`. This is because the `@verbatim` makes the `///` show in the rendered ASCII as well, hence I used a `/**` block instead.

Reply:
Actually, now that I look again how all the other ASCII diagrams are made, this just missed the ``` instead.